### PR TITLE
Add TT token ( testing token )

### PR DESCRIPTION
**Pull request by Timekiller7**
**Description**

Introducing TT token in order to test bridge functionality from sepolia to sepolia-base

**Tests**

No tests, it's only an ERC20 token

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by rcbgr** [comment]
hey @Timekiller7, please list your mainnet contracts too

**Review by wbnns — DISMISSED** [review]
ACK

File: data/TT/data.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (hamdiallam) [inline data/TT/data.json:1]: please fix the json validation errors. Trailing commas for the address keys

**Review by hamdiallam — CHANGES_REQUESTED** [review]

File: data/TT/data.json (hunk 1 above)
Comment (Timekiller7) [reply data/TT/data.json:1]: thank you! fixed

**Comment by hamdiallam** [comment]
Hi @Timekiller7, it looks like the decimals specified here does not match L1 (6 decimals). Can you update this as well?

**Comment by Timekiller7** [comment]
> Hi @Timekiller7, it looks like the decimals specified here does not match L1 (6 decimals). Can you update this as well?

Fixed

**Review by wbnns — APPROVED** [review]
ACK

**Review by annieke — APPROVED** [review]

**Pull request merged**